set.mm

```diff
@@ -613998,6 +614031,150 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
       JZMHZGHABIJZMHZGHCMHZGHADNUAUCGTUBMSBAIEOPPUCUDGUBCMFPPQ $.
   $}
 
+  ${
+    carrydec.a $e |- A e. NN0 $.
+    carrydec.b $e |- B e. NN0 $.
+    carrydec.c $e |- C e. NN0 $.
+    carrydec.1 $e |- ( A + B ) = D $.
+    $( Carry a digit, removing a decimal constructor.  (Contributed by Steven
+       Nguyen, 10-Dec-2022.) $)
+    carrydec $p |- ; A ; B C = ; D C $=
+      ( cdc c1 cc0 cmul co caddc dfdec10 0nn0 dec0u eqid nn0cni addid2i decadd
+      eqtri ) ABCIZIJKIALMZUCNMDCIAUCOAKBCDCUDUCEPFGAEQUCRHCCGSTUAUB $.
+  $}
+
+  ${
+    cdeccl.a $e |- A e. CC $.
+    cdeccl.b $e |- B e. CC $.
+    $( Closure for a numeral.  (Contributed by Steven Nguyen, 11-Dec-2022.) $)
+    cdeccl $p |- ; A B e. CC $=
```

Comment:
Not sure it is very useful, since `; A B` was devised to make sense when A and B are "digits". Maybe @digama0 can tell.

Reply:
I agree with @benjub that this theorem may not be useful.